### PR TITLE
Feat login

### PR DESCRIPTION
**Pull request by parkerpersaud**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,2 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+User.destroy_all
+User.create(email: "squiggles@hey.com", password: "password")
```
Comment (giladshanan) [inline db/seeds.rb:2]: O hey there squiggles

File: spec/features/register_new_user_feature_spec.rb [hunk 2]
```diff
@@ -4,7 +4,6 @@
   scenario 'A student successfully registers' do
     visit '/'
     click_link('sign up')
-    # fill_in "user[username]", with: "Sammy"
     fill_in "user[email]", with: "sammy@cubs.com"
     fill_in "user[password]", with: "password"
     fill_in 'user[password_confirmation]', with: "password"
```
Comment (giladshanan) [inline spec/features/register_new_user_feature_spec.rb:9]: Go Cubs!

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 class SessionsController < ApplicationController
   def new
+
```
Comment (giladshanan) [inline app/controllers/sessions_controller.rb:3]: Is this controller being used?  If not, we might as well delete it.

**Comment by giladshanan** [comment]
I left few little comments, nice work!  It looks like there are quite a few unused devise controllers with commented out actions.   Do we want to keep these around for now in case we want to use them later on?  Or should we delete them?

File: app/controllers/sessions_controller.rb (hunk 3 above)
Comment (parkerpersaud) [reply app/controllers/sessions_controller.rb:3]: Yeah - We can probably get rid of that controller

**Comment by parkerpersaud** [comment]
I think for right now it might be a good idea to keep them since we haven't worked out our two levels of auth. I think we should make sure we address that after we finish up with devise and the different authorizations. We can delete them if we end up not using them.

**Comment by giladshanan** [comment]
Cool!

**Pull request merged**